src/components/Recipes/Recipes.tsx

```diff
@@ -12,14 +12,17 @@ const Recipes: React.FC = () => {
 
   const foodprintCtx = useContext(FoodprintContext);
   const addRecipe: (title: string, id: number, image: string) => void = foodprintCtx.recipes.addRecipe;
+  const itemsReset = foodprintCtx.recipes.itemsReset;
   const ingredientList = foodprintCtx.ingredients.items;
   useEffect(() => {
     if(!isFirstRender) {
+      itemsReset()
       fetchData(addRecipe, ingredientList);
     } else {
       setIsFirstRender(false);
```

Comment:
why do you need this firstRender, useEffect w/ an empty array only calls it on the first render